client/app/scripts/liveblog-edit/directives.js

```diff
@@ -41,8 +43,9 @@ define([
                         originalOrder: 0,
                         pagesManager: new PagesManager($scope.lbPostsBlogId,
                                                        $scope.lbPostsStatus,
-                                                       10,
-                                                       $scope.lbPostsOrderBy || 'editorial'),
+                                                       $scope.lbSticky === 'true'? 100: 10,
```

Comment:
`true` as a string ? Can't we use boolean ?
this makes not a lot of sense to me. Why number of post depends of the sticky posts ?
Could you add comments here to explain why ?
Otherwise we will forget and it could be removed later...